icu4j/main/core/src/main/java/com/ibm/icu/text/MessagePattern.java

```diff
@@ -1382,13 +1382,14 @@ private static int parseArgNumber(CharSequence s, int start, int limit) {
         } else {
             return ARG_NAME_NOT_NUMBER;
         }
-        while (start < limit) {
+        while(start < limit) {
```

Comment:
Why did you remove the spaces after `while` and `if`?
As expected, spotless wants them back.

@echeran FYI

Reply:
I was manually merging them. sorry